### DISCUSSION TITLE
Linter Cop: Auto Download

### DISCUSSION THREAD

**Discussion by Axarath** [StefanMaron/BusinessCentral.LinterCop#278] [General]
I'm facing an issue on several computers. Even if the Auto Download is enabled LinterCop.dll is not updated. LC: Dowload Linter Cop does not work too. Only way how to update it is direct download and copy to the proper directory.

Anyone else has the same issue? Do you know how to solve it?

**Comment by jwikman** [comment]
We've got this issue on Windows 2016 servers, and in those cases it's related to TLS1.2.

I've created a PR to solve that issue, waiting for @StefanMaron to merge.

Maybe you can test that code change and see if that solves your problem as well?

You can find the PR here: https://github.com/StefanMaron/vsc-lintercop/pull/8

**Comment by Arthurvdv** [comment]
@jwikman; Your PR is merged yesterday, hooray!

@Axarath, does this resolve you issue?

**Comment by Axarath** [comment]
@Arthurvdv I tested it and it worked. I'm going to do some more tests but I think it is OK.

**Comment by Arthurvdv** [comment]
Thanks voor de feedback and I'm going to close this conversation.
